### ISSUE TITLE
Can't reliably compile for Apple M1

### ISSUE THREAD

**Issue by grex** [natmlx/NatCorder#3]
tldr: **What is the correct way to enable M1-compatible builds?** 

We're using NatCorder in a Desktop Unity project with a NatML Cloud subscription (username grex).

In order to create a working build that will work on Apple M1 Silicon ...
- we have to change this file `Packages/ai.natml.natcorder/Plugins/macOS/NatCorder.bundle` (i can't seem to find the original forum and/or discord link for this fix)
-  and change its Mac platform setting to `AnyCPU` (from `x86_64`). When compiled without this change, the build will throw a `DllNotFoundException` on launch.

However, the file constantly gets reverted to the `x86_64` value and Unity spits out the following warning: `The package cache was invalidated and rebuilt because the following immutable asset(s) were unexpectedly altered:
  Packages/ai.natml.natcorder/Plugins/macOS/NatCorder.bundle`

- What is the correct way to enable M1-compatible builds? 
- And if it means modifying this file, how do we prevent it from constantly being overwritten?

Our modified NatCorder.bundle:
https://gist.github.com/grex/61e041103d242629a592a9bb0d11f3ce

Diff:
https://www.diffchecker.com/3mEcRjQu

Our NatCorder version is `1.8.4` (also tried `1.8.6` but same problem persists)

**Comment by olokobayusuf** [comment]
Hey @grex what version of Unity are you using? It seems that Unity changed the plugin importer in Unity 2021, so I was able to confirm this issue. I'll publish an update later today with the fix.

The workaround is to manually import NatCorder into your project:
1. In manifest, import the `ai.natml.hub: 1.0.9` package instead.
2. Download NatCorder as a `.zip` from GitHub.
3. Unzip and place the folder in your `Packages/` folder.

Thanks for bringing this to my attention.

**Comment by grex** [comment]
Thank you! Our Unity Version is `2021.3.0f1`

**Comment by olokobayusuf** [comment]
Fixed in [1.8.7](https://github.com/natmlx/NatCorder/releases/tag/1.8.7).
